content/en/docs/setup/production-environment/tools/kops.md

```diff
@@ -23,21 +23,26 @@ a building block.  kops builds on the kubeadm work.
 
 {{% /capture %}}
 
-{{% capture body %}}
+{{% capture prerequisites %}}
 
-## Creating a cluster
+* You must have [kubectl](/docs/tasks/tools/install-kubectl/) installed in order for kops to work.
 
-### (1/5) Install kops
+* The target installation device architecture should be 64-bit.
+
```

Comment:
Q: kops only supports 64-bit architectures?

Reply:
Yes, if you look at the release artifacts [here](https://github.com/kubernetes/kops/releases), all are amd64

Reply:
I recommend clarifying which 64-bit ISAs are acceptable (ie, just AMD64 and Intel 64); there are _lots_ of 64-bit ISAs that aren't compatible with AMD64, and Kubernetes is getting certified on some of them (eg MicroK8s).

(there's also some background in https://github.com/kubernetes/community/blob/master/contributors/design-proposals/multi-platform.md)

Reply:
@kihahu , A minor suggestion: I think it is OK to link to the project page/installation requirements for the project instead of listing the individual 64-bit architectures.
I thought the wording could be more explicit, something like:
```
You must [install](https://github.com/kubernetes/kops#installing) `kops` on a 64-bit device architecture.
```